run_ci.sh

```diff
@@ -8,11 +8,9 @@ function getlogs(){
     # Grab the host journal
     sudo journalctl > $LOGDIR/bootstrap-host-system.journal
 
-    # The logs shared by the ironic containers
-    sudo cp -r /opt/dev-scripts/ironic/log $LOGDIR/container-logs
-
-    sudo podman logs coreos-downloader > $LOGDIR/coreos-downloader.log
-    sudo podman logs ipa-downloader > $LOGDIR/ipa-downloader.log
+    for c in httpd coreos-downloader ipa-downloader ; do
+        sudo podman logs $c > $LOGDIR/$c.log
```

Comment:
What if `podman logs` exits with an error? Should we `|| true` so it keeps going if one fails?

Reply:
yup, will do